### PR TITLE
Adding what's next section to namespace concepts

### PR DESCRIPTION
**Pull request by rajeshdeshpande02**
To learn more about namespaces and give better navigation this section is added.

### CODE REVIEW

File: content/en/docs/concepts/overview/working-with-objects/namespaces.md [hunk 1]
```diff
@@ -112,3 +112,11 @@ kubectl api-resources --namespaced=false
 ```
 
 {{% /capture %}}
+
+{{% capture whatsnext %}}
+* Learn more about [creating a new namespace](/docs/tasks/administer-cluster/namespaces/#creating-a-new-namespace).
+* Learn more about [deleting a namespace](/docs/tasks/administer-cluster/namespaces/#deleting-a-namespace)
+* See [namespaces design](https://github.com/kubernetes/community/blob/{{< param "githubbranch" >}}/contributors/design-proposals/architecture/namespaces.md).
```
Comment (tengqm) [inline content/en/docs/concepts/overview/working-with-objects/namespaces.md]: the "namespaces.md" part is wrong.

Comment (rajeshdeshpande02) [reply content/en/docs/concepts/overview/working-with-objects/namespaces.md]: Thanks @tengqm for your review. I removed that "namespaces.md" part. Can you please have a look again

**Comment by sftim** [comment]
/retitle Adding what's next section to namespace concepts
@rajeshdeshpande02 hope this is is OK?

**Comment by sftim** [comment]
(looks like Prow doesn't know about changing PR titles?)

**Pull request merged**